Apps/odysseus/scripts/test.sh

```
@@ -0,0 +1,51 @@
+}
+trap cleanup EXIT
+
+echo -e "${BLUE}Odysseus Docker Image Test${NC}"
+
+echo -e "${YELLOW}Cloning upstream@${REF}...${NC}"
+git clone --depth 1 --branch "$REF" "$UPSTREAM_REPO" "$WORKDIR" 2>/dev/null \
+  || { git clone "$UPSTREAM_REPO" "$WORKDIR"; git -C "$WORKDIR" checkout "$REF"; }
+
+echo -e "${YELLOW}Test 1: Build image...${NC}"
+docker build -t "$IMAGE" "$WORKDIR" > /dev/null 2>&1 && BUILD_RC=0 || BUILD_RC=$?
+print_test $BUILD_RC "Image builds"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)